versioned_docs/version-latest/monitoring.md

```diff
@@ -0,0 +1,558 @@
+
+**Prerequisites**
+
+Small cloud server with the minimum spec. 1/2/20 for monitoring is recommended.
+
+## 1. Intsalling Grafana Server.
+
+
+### Download
+```bash
+sudo apt-get install -y apt-transport-https software-properties-common wget
+sudo mkdir -p /etc/apt/keyrings/
+wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+echo "deb [signed-by=/etc/apt/keyrings/grafana.gpg] https://apt.grafana.com stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+```
+Update the list of available packages:
+```
+sudo apt-get update
+```
+Install the latest Enterprise release:
+```
+sudo apt-get install grafana-enterprise
+```
+*this also will do this postinstall script*:
+> addgroup --system grafana
+adduser --system --home /usr/share/grafana --no-create-home --ingroup grafana --disabled-password --shell /bin/false grafana
+
+> Plus is gives permissions for system user grafana to some folders.
+
+
+
+### Start Server
+
+```sudo systemctl daemon-reload
+sudo systemctl start grafana-server
+sudo systemctl status grafana-server
+
+sudo systemctl daemon-reload
+sudo systemctl start grafana-server
+sudo systemctl status grafana-server
+```
+### Config
+```
+cd /etc/grafana
+sudo vi garafana.ini
+http_port=12345			# This is example. Here specify your custom SSH port (if set)
+```
+### Enter your Grafana Server
+
+Get IP on your cloud server:
+`ip -4 addr`  
+From home PC:  
+```
+http://your-cloud-server-IP-address:12345
+```
+
+**In "Connections" section specify the same port as you will do in Prometheus.**
+
+### Setup Jobs
+
+```
+job:
+/subspacenode|subspaceserver/
+```
+:::note
+The Dashboard for scraping metrics from Subspace Node is available on this [URL](https://github.com/counterpointsoftware/subspace-monitoring/blob/main/grafana/provisioning/dashboards/subspace-dashboard-counterpoint.json).  
+To add it to your Grafana server just copy-paste the content.
+:::
+:::note
+For general server monitoring you can DL popular dashboard for Nodeexporter, named: Node Exporter full (most rated)
+:::
+
+
+-----------------------------------------------------------------------------------------------------------------------------
+## 2. Installing Prometheus
+
+Command-line flags configure immutable system parameters (such as storage locations, amount of data to keep on disk and in memory, etc.) 
+Configuration file (yml) defines everything related to scraping jobs and their instances, as well as which rule file to load. Canonically should be in `/etc/prometheus`  
+To view all available command-line flags, run `./prometheus -h`
+
+The following commands must be ran as root.
+
+### Create Group and User
+```
+sudo addgroup --system prometheus
+sudo useradd -M -r -g prometheus -s /sbin/nologin prometheus
+```
+> > -M: do not create user's home directory;	-r: create a system user;	-s /sbin/nologin: set user's shell as 'nologin';	-g prometheus - add to group 'prometheus'
```

Comment:
Is the `> >` to add a quote within a quote what we actually want here? Feels like it should be a tip block or similar?